### PR TITLE
(DOC): basic usage docs

### PR DESCRIPTION
**Pull request by matklad**
@alexeykudinkin @atsky I've tried to write some Rust code (my homework :) ) in intellij rust yesterday, and it is possible :)

So I started some documentation for the users of the plugin.

Guys, what do you think? Isn't it too early?

### CODE REVIEW

File: doc/Usage.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Installation
+
```
Comment (alexeykudinkin) [inline doc/Usage.md]: This seems like a candidate for a migration right into `README.md`. Thoughts?

**Comment by alexeykudinkin** [comment]
:+1:

**Comment by atsky** [comment]
Super!

File: doc/Usage.md (hunk 1 above)
Comment (matklad) [inline doc/Usage.md]: Yes, I'll move it

**Pull request merged**